plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -165,6 +231,49 @@ def disruption_budget_target(deployment)
       end
     end
 
+    def add_env_config_map
+      return unless env_from_config_map?
+      return if env_config_map_exists?
+
+      annotations = reference_annotations.
```

Comment:
should be fine to use `primary` here and leave add_pod_disruption_budget as it was